api/v1beta1/types.go

```diff
@@ -920,6 +925,11 @@ type VCN struct {
 	// +listType=map
 	// +listMapKey=name
 	NetworkSecurityGroups []*NSG `json:"networkSecurityGroups,omitempty"`
+
+	// DnsLabel specifies a DNS label for the VCN, used in conjunction with the VNIC's hostname and
+	// subnet's DNS label to form a fully qualified domain name (FQDN) for each VNIC
+	// within this subnet (for example, `bminstance1.subnet123.vcn1.oraclevcn.com`).
+	DnsLabel *string `json:"dnsLabel,omitempty"`
```

Comment:
Should this be optional?

Reply:
Definitely, will fix it

Reply:
looks like omitempty has the same result, but added optional as well just to be sure.